src/main/java/services/update_entities/UpdateEvent.java

```diff
@@ -0,0 +1,32 @@
+package services.update_entities;
+
+import data_gateway.CalendarManager;
+import data_gateway.EventEntityManager;
+
+import java.time.LocalTime;
+
+public class UpdateEvent {
+    CalendarManager calendarManager;
+    long id;
+
+    public UpdateEvent(EventEntityManager eventEntityManager, long id){
+        this.calendarManager = eventEntityManager;
+        this.id = id;
+    }
+
+    public void updateName(String newName){
+        calendarManager.updateName(id, newName);
+    }
+
+    public void updateStartTime(LocalTime startTime){
```

Comment:
Since these are actually not in use right now, how about adding a new option to the ApplicationDriver to show a basic usage of editing?